Header header          #standard header information

NodeMsg[] nodes
FactorMsg[] factors

std_msgs/Bool use_submap
std_msgs/Float64 interchange_radius
std_msgs/Float64 compound_radius
std_msgs/Float64 min_keyframe_dist
std_msgs/Float64 min_keyframe_rot_deg
std_msgs/Bool use_keyframe